packages/features/test/e2e/features.e2e.test.ts

```diff
@@ -1,8 +1,7 @@
-import { FeatureService, ICollection } from '../../src/index';
+import 'isomorphic-fetch';
+import { FeatureService, ICollection } from '../../src/features';
 
-require('isomorphic-fetch');
-
-const TEST_SITE = 'https://demo.ldproxy.net/vineyards';
+const TEST_SITE = 'http://localhost:5000';
```

Comment:
For sure, I will update the README.

Reply:
I have attached the instructions to run E2E tests locally to the root README of this library, because those instructions apply to both features and processes.